### PR TITLE
Lint

### PR DESCRIPTION
**Pull request by yakov116**
I hope to do some https://github.com/refined-github/refined-github/issues/5222 too

### CODE REVIEW

**Comment by yakov116** [comment]
Don't merge this the icons are messed up. I need to look into it

File: source/features/sticky-csv-header.css [hunk 1]
```diff
@@ -13,8 +13,8 @@
 
 .blob-wrapper.type-csv,
 .Box:has(.csv-data) {
-	border-bottom-right-radius: 0;
-	border-bottom-left-radius: 0;
+	border-bottom-right-radius: none;
+	border-bottom-left-radius: none;
```
Comment (FloEdelmann) [inline source/features/sticky-csv-header.css:17]: This is not allowed, see https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius

Also see https://github.com/xojs/stylelint-config-xo/pull/17, probably you have an old version locally.

**Review by FloEdelmann — CHANGES_REQUESTED** [review]

File: source/features/sticky-csv-header.css (hunk 1 above)
Comment (fregante) [reply source/features/sticky-csv-header.css:17]: That's explain why CI wasn't failing. 👌 

Feel free to "restore the file" once you accept the membership 😉

Comment (FloEdelmann) [reply source/features/sticky-csv-header.css]: Both done. Thanks for inviting me :)

**Pull request closed without merging**